### PR TITLE
Initial configuration of bots

### PR DESCRIPTION
**Pull request by tiinanguyen**
Replicates [iOS configuration PR](https://github.com/RevenueCat/purchases-ios/pull/480). All links are relevant to this SDK (e.g: links to existing issues and CONTRIBUTING.md file). Created a CONTRIBUTING.md file for this repo.

### CODE REVIEW

**Review by tiinanguyen — COMMENTED** [review]
@vegaro any clue why 'ci/circleci: android' is failing?

**Comment by vegaro** [comment]
Idk why that would start failing now. I am taking a look

**Comment by vegaro** [comment]
I just reran it and it works :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
-    1. Platform:
-    2. SDK version:
-    3. OS version:
-    4. XCode/Android Studio version:
-    5. How widespread is the issue. Percentage of devices affected.
+   1. Platform:
+   2. SDK version:
+   3. OS version:
+   4. XCode/Android Studio version:
+   5. How widespread is the issue. Percentage of devices affected.
```
Comment (vegaro) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: Should we specifically ask for React Native version somewhere?

Comment (tiinanguyen) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: We can! Do you think React Native version impacts certain issues?

Comment (vegaro) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: Yes, it definitely does

Comment (aboedo) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: +1 for RN version.
Also, for this one devs can install our SDK either through cocoapods or manually, so maybe specifying if they're using cocoapods, and if so, version number

Comment (tiinanguyen) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Sounds good, just added the changes! Would be a good signal for developers to update their versions

**Review by vegaro — APPROVED** [review]

**Pull request merged**